doc/mgr/nfs.rst

```diff
@@ -128,6 +138,17 @@ Delete NFS Ganesha Cluster
 
 This deletes the deployed cluster.
 
+Since the removal of NFS daemon(s) and the ingress service (if deployed) is
+asynchronous, check the statuses of the NFS and ingress services to make sure
+that they have been deleted successfully. If your orchestrator is cephadm,
+execute the following and check whether no services are reported:
```

Comment:
Suggest "execute the following to check that the services are no longer reported."

Reply:
Done! Thanks!